package.json

```diff
@@ -46,16 +46,17 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jest": "^21.2.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.4.0",
-    "webpack": "^3.7.1"
+    "eslint-plugin-react": "^7.4.0"
   },
   "peerDependencies": {
-    "eslint": ">=3"
+    "eslint": ">=3",
+    "webpack": ">=3"
```

Comment:
If we make webpack a peer dependency then all projects which are not using the webpack part of the config will log an error because they don't have webpack. I think I'm fine removing webpack from the dependencies, but I don't want it in the peerDependencies.